src/index.js

```diff
@@ -0,0 +1,5 @@
+import types from './types'
+import infer from './infer'
+import Schema from './schema'
+
+module.exports = { infer, types, Schema }
```

Comment:
I did not see the reason to expose all the type for public use. TypeGuesser it is kind of wrapper, and only it should be available publicly, I think